30413/Bogdan Emilian Sandu/Lab07/README.md

```diff
@@ -1,5 +1,5 @@
-# Lab X
-
-> Edit this file and describe in a few words what you've done in this lab.
+# Lab 07
 
+>Learned about graphs, how to implement them and how to traverse them using depth first search and breadth first search.
+>Solved the extra assignment. I think I can do it faster but I didn't have time.
```

Comment:
Explain the strategy you used for solving the extra problem.

Reply:
Just updated it.

Reply:
Good approach!

It would have been easier to update all the nodes adjacent to a node you selected and remove the inbound edges.

This way, at each step you could select any node that has the inner degree 0.

Anyway, your approach is valid, well done!